modules/data-entry/src/main/java/ca/sickkids/ccm/lfs/dataentry/internal/serialize/labels/AnswerOptionsLabelProcessor.java

```diff
@@ -47,7 +47,10 @@
     public void leave(Node node, JsonObjectBuilder json, Function<Node, JsonValue> serializeNode)
     {
         try {
-            if (node.isNodeType("lfs:TextAnswer")) {
+            if (node.isNodeType("lfs:TextAnswer")
+                || node.isNodeType("lfs:LongAnswer")
+                || node.isNodeType("lfs:DoubleAnswer")
+                || node.isNodeType("lfs:DecimalAnswer")) {
```

Comment:
I think this should work just as fine:

```suggestion
            if (node.isNodeType("lfs:Answer")) {
```

But, with this simpler approach (and possibly with your original approach), it is possible that it would override another label processor. A safer check would be to also check here if there is indeed a question, and the question has answer options, before we proceed with this.

Reply:
In particular, I think this may randomly conflict with booleans, and with the more generic approach, it may conflict with everything else (pedigree, files, dates...).